src/toil/test/wdl/wdltoil_test_kubernetes.py

```diff
@@ -51,7 +51,7 @@ def test_wdl_kubernetes_cluster(self):
         self.leader = self.cluster.getLeader()
 
         url = "https://github.com/DataBiosphere/wdl-conformance-tests.git"
-        commit = "09b9659cd01473e836738a2e0dd205df0adb49c5"
+        commit = "7c7f82c0dd426f5be708f7a858e448cfb0fea91b"
```

Comment:
Why is this commit hash different from the one in `src/toil/test/wdl/wdltoil_test.py`?

Reply:
I've changed the commit for both tests to come from just one place.